spec/features/trainer_spec.rb

```diff
@@ -133,6 +138,7 @@
     before do
       create(:user_with_two_blocks_without_cards)
       visit trainer_path
+      click_link 'ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.